lib/index.js

```diff
@@ -0,0 +1,8 @@
+var fs = require('fs')
+
+var riot = module.exports = require('./compiler')
+
+require.extensions['.tag'] = function(module, filename) {
+  var src = riot.compile(fs.readFileSync(filename, 'utf8'))
+  module._compile(src, filename)
```

Comment:
So it's possible to have multiple tags on the same file? ie. `require('./tag-library.tag')` ?